src/seq/mod.rs

```diff
@@ -292,6 +292,10 @@ pub trait IteratorRandom: Iterator + Sized {
     /// available, complexity is `O(n)` where `n` is the iterator length.
     /// Partial hints (where `lower > 0`) also improve performance.
     ///
+    /// Note that value stability depends on the size hints. In particular,
```

Comment:
I would say "value chosen" or "index chosen" rather than "value stability"

Reply:
"Value stability" is a term we've used quite a lot when talking about reproducibility of output, at least internally (it doesn't appear in documentation much). But, yes, we can probably reduce usage of jargon here. Maybe we should say:

> Note that the output value and the number of RNG samples used depends on size hints. ...

Reply:
That sounds very clear to me 👍